openwisp_network_topology/settings.py

```diff
@@ -2,7 +2,10 @@
 
 from django.conf import settings
 
-logger = logging.getLogger(__name__)
+# The logger 'pre_django_setup' is suppose to be
```

Comment:
is supposed?

Reply:
Didn't know it's supposed to be `supposed` when `suppose` is used phrase in a case like this. Changed it.